server/gameconnection.py

```diff
@@ -294,44 +298,76 @@ async def handle_game_result(self, army, result):
         except (KeyError, ValueError):  # pragma: no cover
             self._logger.warning("Invalid result for %s reported: %s", army, result)
 
-    async def handle_operation_complete(self, army, secondary, delta):
-        if not int(army) == 1:
+    async def handle_operation_complete(
+        self, primary: Any, secondary: Any, delta: str
+    ):
+        """
+        :param primary: are primary mission objectives complete?
+        :param secondary: are secondary mission objectives complete?
+        :param delta: the time at which the operation occurred
```

Comment:
This would be how long it took to complete the mission. That's why whoever named the parameter called it `delta`, because it's an amount of time, not a timestamp.